src/Frontend.elm

```diff
@@ -1505,166 +1581,167 @@ myGroupsView userConfig model loggedIn =
+searchInput { theme, texts } searchText =
+    Input.text
+        [ width <| maximum 400 fill
+        , Border.rounded 5
+        , Border.color theme.darkGrey
+        , paddingEach { left = 24, right = 8, top = 4, bottom = 4 }
+        , Background.color theme.background
         , Ui.onEnter SubmittedSearchBox
-        , Dom.idToAttribute groupSearchId |> Element.htmlAttribute
-        , Element.inFront
-            (Element.el
-                [ Element.Font.size 12
-                , Element.moveDown 6
-                , Element.moveRight 4
-                , Element.alpha 0.8
-                , Element.htmlAttribute (Html.Attributes.style "pointer-events" "none")
+        , Dom.idToAttribute groupSearchId |> htmlAttribute
+        , inFront <|
+            el
+                [ Font.size 12
+                , moveDown 6
+                , moveRight 4
+                , alpha 0.8
+                , htmlAttribute (Html.Attributes.style "pointer-events" "none")
                 ]
-                (Element.text "🔍")
-            )
+                (text "🔍")
         ]
         { text = searchText
         , onChange = TypedSearchText
         , placeholder = Nothing
-        , label = Element.Input.labelHidden "Search for groups"
+        , label = Input.labelHidden texts.searchForGroups
         }
 
 
 searchInputLarge : UserConfig -> String -> Element FrontendMsg
-searchInputLarge userConfig searchText =
-    Element.row
-        [ Element.width <| Element.maximum 400 Element.fill
-        , Element.centerX
+searchInputLarge { theme, texts } searchText =
+    row
+        [ width <| maximum 400 fill
+        , centerX
         ]
-        [ Element.Input.text
-            [ Element.Border.roundEach { topLeft = 5, bottomLeft = 5, bottomRight = 0, topRight = 0 }
-            , Element.Border.color userConfig.darkGrey
-            , Element.Border.widthEach { bottom = 1, left = 1, right = 0, top = 1 }
-            , Element.paddingEach { left = 30, right = 8, top = 8, bottom = 8 }
+        [ Input.text
+            [ Border.roundEach { topLeft = 5, bottomLeft = 5, bottomRight = 0, topRight = 0 }
+            , Border.color theme.darkGrey
+            , Border.widthEach { bottom = 1, left = 1, right = 0, top = 1 }
+            , paddingEach { left = 30, right = 8, top = 8, bottom = 8 }
             , Ui.onEnter SubmittedSearchBox
-            , Element.Background.color userConfig.background
-            , Dom.idToAttribute groupSearchLargeId |> Element.htmlAttribute
-            , Element.inFront
-                (Element.el
-                    [ Element.Font.size 14
-                    , Element.moveDown 9
-                    , Element.moveRight 6
-                    , Element.alpha 0.8
-                    , Element.htmlAttribute (Html.Attributes.style "pointer-events" "none")
+            , Background.color theme.background
+            , Dom.idToAttribute groupSearchLargeId |> htmlAttribute
+            , inFront <|
```

Comment:
This repo already breaks this rule but avoid using `<|`. Stick to using parenthesis or `|>` (I should add https://package.elm-lang.org/packages/SiriusStarr/elm-review-pipeline-styles/latest/ to the elm-review config). My reasoning for this is that `<|` doesn't look good if there's line breaks (especially if there's multiple `<|`) and it's entirely redundant so sticking with `|>` and parenthesis reduces the amount of decisions that need to be made.

Reply:
Alright 👍